File: server/src/main/kotlin/models/RaceInfo.kt
package models

data class RaceInfo(
    val bestLapNum: List<Int>,
    val bestLapTime: List<Float>,
    val carClass: List<Int>,
    val carClassPosition: List<Int>,
    val raceTime: List<Float>,
    val lapsCompleted: List<Int>,
    val percentageAroundTrack: List<Float>,
    val lastLapTime: List<Float>,
    val onPitRoad: List<Boolean>,
    val position: List<Int>,
    val onTrackStatus: List<Int>
)

File: classes/RaceInfo.py
from dataclasses import dataclass
from typing import List

"""
Info about the current state of the race.
"""


# Time series data (tick)
@dataclass
class RaceInfo:
    bestLapNum: List[int]
    bestLapTime: List[float]
    carClass: List[int]
    carClassPosition: List[int]
    raceTime: List[float]
    lapsCompleted: List[int]
    percentageAroundTrack: List[float]
    lastLapTime: List[float]
    onPitRoad: List[bool]
    position: List[int]
    onTrackStatus: List[int]